core/server/config/defaults.json

```diff
@@ -19,6 +19,9 @@
     "scheduling": {
         "active": "SchedulingDefault"
     },
+    "members": {
+        "payment_processors": []
```

Comment:
99% of our config values use camel case :)
I see that e.g. spam config is not camel case. We should change that in v3.